controller/rest/dlp_rule.go

```diff
@@ -1124,6 +1166,40 @@ func handlerDlpRuleConfig(w http.ResponseWriter, r *http.Request, ps httprouter.
 	restRespSuccess(w, r, nil, acc, login, &rconf, "Edit dlp rule")
 }
 
+func processGroupDlpSensors(w http.ResponseWriter, cg *share.CLUSDlpGroup, sensors []api.RESTDlpConfig) error {
```

Comment:
Maybe a unit test for this function?

Reply:
I have conducted unit tests using 2 federated cluster, this function is tested, thanks

Reply:
To make sure we're on the same page, when saying unit tests, I mean something like this: https://github.com/neuvector/neuvector/blob/main/share/utils/utils_test.go

Reply:
After discussed offline, we can do this later.

Reply:
I will create a separate jira case to write test cases for DLP/WAF since it can take quite some time, thanks